### PR TITLE
Completed the deliverable - Fatima's pod

### PR DESCRIPTION
**Pull request by monicaleep**
Completed the deliverable.

### CODE REVIEW

**Comment by monicaleep** [comment]
My name Monica Parrillo - My point person is Fatima

File: solution.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+console.log('Exercise 5 Result:\n', foods);
+
+/*
+Exercise 6:
+  - Replace the string 'pizza' in the foods array with the strings 'sushi' & 'cupcake'.
+*/
+
+// Complete Exercise 6 below...
+
+foods.splice(foods.indexOf('pizza'),1,'sushi','cupcake');
```
Comment (ghost) [inline solution.txt:63]: nice use of a couple of array methods here! you've done it programmatically, so no matter where `pizza` is in the array, you'll always splice it out 👍

File: solution.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+var total=0;
+// Complete Exercise 15 below...
+numArrays.forEach(numArray=>{
+  numArray.forEach(num=>{
+    total += num;
+  })
+})
+
+
+console.log('Exercise 15 Result:\n', total);
```
Comment (ghost) [inline solution.txt:205]: nice! super clean use of nested `forEach()` 👍

**Comment by ghost** [comment]
awesome work! marking complete, feel free to close out PR

**Pull request closed without merging**